### PR TITLE
docs(changelog) add changelog.md

### PR DESCRIPTION
**Pull request by takam5f2**
Signed-off-by: Takayuki AKAMINE <takayuki.akamine@tier4.jp>

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Supported OS: Ubuntu 20.04
+- Supported ROS Dist.: ROS Galactic
+- Improved output message from `path.verify()` method
+- Improved view of graphs
+- Added wildcard support for `Architecture.callbacks()` method
+- Added command line function to check trace data
+- Fixed specification of `Architecture.search_paths()` method
+- Added function to extract duplicated timer callback
+
+### v0.2.3 <small>\_ July 14, 2022</small> {id = "2.2.0"}
```
Comment (hsgwa) [inline docs/changelog.md:16]: Is this a mistake for v0.2.2?

**Review by hsgwa — COMMENTED** [review]
Let me confirm the content of changelog.

File: docs/changelog.md (hunk 1 above)
Comment (takam5f2) [reply docs/changelog.md]: @hsgwa 
resolved at https://github.com/tier4/CARET_doc/pull/69/commits/751ce41bd38ee9ed3392d361cf0c4c328088a924

**Review by hsgwa — APPROVED** [review]
LGTM

**Pull request merged**